config/config.yaml

```diff
@@ -0,0 +1,330 @@
+# configuration file for ibf drought pipeline
+
+# Metadata for Drought Pipeline Configuration
+# --------------------------------------------
+# Author: [Your Name or Organization]
+# Created: [Date]
+# Last Updated: [Date]
+# Purpose: This configuration file is used to define parameters for the IBF drought pipeline, including country-specific thresholds, ensemble member settings, and seasonal lead times for Early Warning Systems.
+# Usage: The file is read by the IBF data pipeline to execute drought risk analyses and determine early action triggers.
+# Notes:
+# - Trigger model,Trigger levels and probabilities are defined per country and season.
+# - Lead times are specified for each month, grouped by country and season.
+
+# Key Components:
+# - `countries`: Country-specific configurations for drought risk analysis, including trigger models, thresholds, and seasonal lead times.
+
+
+droughts: 
+  no_ensemble_members: 51  # number of ensemble members to consider
+  minimum_for_drought_extent: 0.01  # minimum probability for drought extent 
+ 
+
+auxiliary:
+  worldpop_url: https://data.worldpop.org/GIS/Population/Global_2021_2022_1km_UNadj/constrained/2022
+
+databases:
+  blob_container: ibfdatapipelines
+  blob_storage_path: drought
+  postgresql_server: 510-postgresql-flex-server.postgres.database.azure.com
+  postgresql_port: 5432
+  postgresql_database: global510
+
+countries:
+  - name: KEN
+    admin-levels:
+      - 1
+    trigger-on-lead-time: 3 
+    pipeline-will-trigger-portal: disable
+    classify-alert-on: disable
+    alert-on-minimum-probability:
+      min: 0.65
+      med: 0.75
+      max: 0.85
```

Comment:
just to check. This represents the potential for multi-threshold you set up. In practice this is not used for any of the countries right?

Reply:
it is disabled for now and can be enabled latter